delta-app/src/main/java/io/cdap/delta/app/DeltaWorker.java

```diff
@@ -394,19 +400,27 @@ private void applyEvent(Sequenced<? extends ChangeEvent> event) throws Exception
   }
 
   private void startFromLastCommit() throws Exception {
-    eventQueue.clear();
```

Comment:
It seems like the clear() method will atomically removes all of the elements in the queue. Can we have a more detailed example here? I am not too sure what's the difference here?

Reply:
Got it. Thanks for the comments.